zips/draft-ecc-community-and-coinholder.md

```diff
@@ -66,7 +65,6 @@ It would immediately increase coinholders’ voice, minimize governance confusio
 * The funds in the Deferred Dev Fund Lockbox will be usable immediately on activation of this ZIP.
 * Any use of Deferred Dev Fund Lockbox funds is consistent with the purpose specified in [^zip-1015-lockbox] of "funding grants to ecosystem participants".
 
-
```

Comment:
It's not important (and makes no difference to rendering), but we generally leave two lines between top-level sections.